src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.module.scss

```
@@ -0,0 +1,57 @@
+@import "../../../../../tokens";
+
+.wrapper {
+  width: 100%;
+  height: 100%;
+  background-color: $color-grey-05;
+}
+
+.dashboardContent {
+  padding-block: $spacing-2xl;
+  padding-inline: $spacing-xl;
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-lg;
+}
+
+.exposuresArea {
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-sm;
+
+  .exposuresAreaHeadline {
+    font: $text-body-xl;
+    font-weight: 600;
+  }
+
+  .exposuresAreaDescription {
+    font: $text-body-md;
+  }
+
+  .exposuresFilterWrapper {
+    padding-block: $spacing-md;
+  }
+}
+
+.exposureList {
+  list-style-type: none;
+  padding-inline: 0;
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-lg;
+}
+
+.visuallyHidden {
+  // These styles are taken from
+  // https://react-spectrum.adobe.com/react-aria/VisuallyHidden.html
+  border: 0;
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  margin: 0 -1px -1px 0;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  white-space: nowrap;
+}
```

We already have `@mixin visually-hidden` in `tokens.scss` and I also saw you it already using [here](https://github.com/mozilla/blurts-server/pull/3211/files#diff-7019c487730a4e636d2ee05fe8062186a8442095d864250bbadbd59340f5f06eR135-R137). :)

```suggestion
.visuallyHidden {
  @include visually-hidden;
}
```